helpers/inserts.js

```diff
@@ -21,7 +21,7 @@ function Inserts (record, values, concat) {
 
 	var self = this;
 
-	if (!record || !record.table || !record.values) {
+	if (!record || !record.table || !record.fields) {
```

Comment:
Why change behaviour here from checking if fields property exists instead of values?

Reply:
Change is correct.